src/components/item/MovieItem.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import Col from 'react-bootstrap/Col';
+import styles from './styles.module.css';
+
+const MovieItem = (props) => {
+    const { id, title, genres, poster_path, release_date } = props.movie;
+    const year = new Date(release_date).getFullYear();
+    const genre = genres.join(', ');
+
+    return (
+        <Col md={4} className={styles.serialWrapper}>
+            <a href="#">
+                <img className={styles.serialPoster} src={poster_path} alt={title} />
+                <div className={styles.serialMeta}>
```

Comment:
Starting from this \<div\>, this block of elements would better be extracted into separate dumb component, e.g. `MovieInfo` from semantic point of view.